### DISCUSSION TITLE
The osd pod restart need more than 8 min, why?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12326] [Q&A]
The osd is HDD.
ceph version is v16.2.13

**Comment by sp98** [comment]
It should not take 8 minutes. 
Whats causing the osd pod restarts? Cluster upgrade? 
Can you share the operator logs?

**Reply by zhucan** [reply]
Because of there is slow ops on the osd, so I delete the pod manually. @sp98

**Comment by travisn** [comment]
What do the events on the osd pod show? `kubectl describe pod <osd>`
Look at the logs for the init containers and see which step is taking a long time

**Comment by zhucan** [comment]
When restating the osd again, I will show the logs
